examples/src/main/scala/org/apache/spark/examples/ml/OneVsRestExample.scala

```diff
@@ -155,7 +155,7 @@ object OneVsRestExample {
 
     // evaluate the model
     val predictionsAndLabels = predictions.select("prediction", "label")
-      .map(row => (row.getDouble(0), row.getDouble(1)))
+      .rdd.map(row => (row.getDouble(0), row.getDouble(1)))
```

Comment:
we should either put `.rdd` at the end of the previous line or make a new line for it